### DISCUSSION TITLE
Retrieving Project Name

### DISCUSSION THREAD

**Discussion by DMW9400** [leolabs/ableton-js#100] [Q&A] [answered]
Hello all,
@leolabs  or anyone else with any experience on this matter - is it possible to pull the project name anywhere from the live API? I've been looking into the issue and it seems like most of the solutions are pretty hack-y, like pulling from Ableton's log files, adding max extensions, etc.

**Comment by leolabs** [comment, marked as answer]
Unfortunately, I haven't found a way to do this either. Your best bet would be to parse it out of the log files if you want to support all editions of Live.

**Comment by DMW9400** [comment]
I actually haven't been able to find the file name in the log either! I can't find any logs at the /Users/[username]/Library/Preferences/Ableton/Live x.x.x/Log.txt filepath Ableton specifies (for mac) in its documentation. And the log I can find (in User/[username]/Library/Application Support/Ableton/Live Reports/usage) doesn't include the file name. Am I missing something? Many thanks for all your help and perspective so far!

**Comment by leolabs** [comment]
Hmm, there should be logs in the first path you mentioned, that's what I use as well. Have you tried manually navigating to the path in Finder and checking if you can find the log files that way?
